packages/lib/models/Setting.test.ts

```diff
@@ -273,6 +273,23 @@ describe('models/Setting', function() {
 		expect(Setting.value('style.editor.contentMaxWidth')).toBe(600); // Changed
 	}));
 
+	it('should migrate to new setting', (async () => {
+		await Setting.reset();
+
+		Setting.setValue('spellChecker.language', 'fr-FR');
+		Setting.applyUserSettingMigration();
+		expect(Setting.value('spellChecker.languages')).toStrictEqual(['fr-FR']);
+	}));
+
+	it('should not override new setting, if it already set', (async () => {
+		await Setting.reset();
+
+		Setting.setValue('spellChecker.languages', ['fr-FR', 'en-US']);
+		Setting.setValue('spellChecker.language', 'fr-FR');
+		Setting.applyUserSettingMigration();
+		expect(Setting.value('spellChecker.languages')).toStrictEqual(['fr-FR', 'en-US']);
+	}));
```

Comment:
Also please add a test when "spellChecker.language" is not set to any value.

Reply:
Added.